pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -417,25 +417,25 @@ load 5m
   mins   0   10 0  30 0   -10  0  5   NaN 0   NaN NaN
   maxes  10  20 5  0  -10 -5   40 50  10  NaN NaN NaN
   series -10 0  10 20 30  40   50 NaN 60  70  80  NaN
-  mixed_metric {{schema:0 sum:5 count:4 buckets:[1 2 1]}} 1 2 3 {{schema:0 sum:5 count:4 buckets:[1 2 1]}} {{schema:0 sum:8 count:6 buckets:[1 4 1]}}
+  mixed_metric {{schema:0 sum:5 count:4 buckets:[1 2 1]}} 1 2 3
```

Comment:
why is this dropped?

Reply:
I think I was trying different things to update the test case and forgot to add this back after making the other changes. I'll restore it.

Reply:
Resolved in d457cc1